content/howto/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Development"
+title: "How-Tos"
```

Comment:
@osterman I included the new, empty sections to get the structure down. Do you think we should keep them hidden until we actually have content in here? Should we possibly setup a staging site prior to merging this full reorg / v2 of the docs? Something to discuss -- Let me know your thoughts.

Reply:
Let's keep them for now